### DISCUSSION TITLE
Notice limits - maximum number of lots, notice file size

### DISCUSSION THREAD

**Discussion by kpankowski** [OP-TED/eForms-SDK#936] [Q&A] [answered]
Hi,

I would like to inquire about the current limits regarding eForms notices:

1. Is there a limit for the number of lots that can be created in a single notice?
2. Is there a limit for the file size that can be submitted?

I apologize if this information has already been discussed or if it is available in the documentation, but I have only found information dated September 30, 2022, stating that there was a 3MB limit per notice. Could you please confirm if this limit is still valid?

Regards,
KP.

**Comment by YvesJo** [comment, marked as answer]
Hi,
For this question, please get directly in touch with the helpdesk.
KR

**Reply by kpankowski** [reply]
Okay, I will try. Unfortunately, I have never received any response when using the helpdesk form so far.
